src/common/components/virtualized/tableCell.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+
+// "originalColumn" will remain as <any> (same like react-virtualized) but it is actually a React node.
```

Comment:
I left it as _any_ because I didn't find that type so thanks to your comment I found it: `React.ReactElement<>`. I'll change some typings.